### PR TITLE
Create additional test data

### PR DESCRIPTION
**Pull request by martin-fritz**
A bunch of new Assets (as built/as planned), Quality alerts and quality investigations.

### CODE REVIEW

**Review by hgo-ds — COMMENTED** [review]
The new assets, both asPlanned and asBuild, have no further parent/child relations, is that how it should be?

File: tx-backend/src/main/resources/db/testdata/afterMigrate__020_asset_as_planned.sql [hunk 1]
```diff
@@ -3,15 +3,35 @@
...
-    (${assetAsPlannedId09}, 'Audi-A7'         , 'A7XXX'             , 'A7'                     , 'OK'        , 'C-Level'     , 'CUSTOMER', 'SERIALPART'       , 'production', 'Audi AG'        , '--', '9923399774344455937394339', 'urn:uuid:bf23b459-d3cc-445e-8839-3fe7f49d8cc5', '2019-02-04T10:00:00', '2025-02-04T10:00:00', '2022-02-04T08:00:00', '2024-02-04T08:00:00'),
-    (${assetAsPlannedId10}, 'Audi-A8'         , 'A8XXX'             , 'A8'                     , 'OK'        , 'C-Level'     , 'CUSTOMER', 'SERIALPART'       , 'production', 'Audi AG'        , '--', '5422724375732744449977947', 'urn:uuid:28ddd434-fabc-4e82-bfe9-d2aab5d64251', '2019-02-04T10:00:00', '2025-02-04T10:00:00', '2022-02-04T08:00:00', '2024-02-04T08:00:00');
+    (${assetAsPlannedId01}, 'H-LeftTailLight'    , 'XT2309'            , 'Xenon Left-Taillights'      , 'OK'        , 'C-Level'     , 'OWN'     , 'PARTASPLANNED'       , 'production', 'Hella'          , '--', '5739525733343254979259249', 'BPNL000000000001', '2019-03-04T10:00:00', '2025-03-04T10:00:00', '2022-02-04T08:00:00', '2024-02-04T08:00:00'),
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__020_asset_as_planned.sql:8]: To my understanding, the very same users would manage asBuilt and asPlanned assets at the same time. So it's a more realistic setup to use the same BPNs for both.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__034_received_alert_on_as_built_closed.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+-- join alert to asset
+insert into assets_as_built_alerts
+    (alert_id           , asset_id)
+values
+    (${alertReceivedId5}, ${assetAsBuiltId22});
+
+---
+update assets_as_built
+    set active_alert = true
+    where id in (${assetAsBuiltId22});
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__034_received_alert_on_as_built_closed.sql:41]: Correct, should not be set to true for closed/cancelled alerts and investigations.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__045_sent_alert_on_as_built_canceled.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+-- join alert to asset
+insert into assets_as_built_alerts
+    (alert_id   , asset_id)
+values
+    (${alertSentId6}, ${assetAsBuiltId15});
+
+---
+update assets_as_built
+    set active_alert = true
+    where id in (${assetAsBuiltId15});
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__045_sent_alert_on_as_built_canceled.sql:41]: Correct, should not be set to true for closed/cancelled alerts and investigations.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__046_sent_alert_on_as_built_closed.sql [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+-- join alert to asset
+insert into assets_as_built_alerts
+    (alert_id       , asset_id)
+values
+    (${alertSentId7}, ${assetAsBuiltId15});
+
+---
+update assets_as_built
+    set active_alert = true
+    where id in (${assetAsBuiltId15});
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__046_sent_alert_on_as_built_closed.sql:41]: Correct, should not be set to true for closed/cancelled alerts and investigations.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__065_sent_investigation_on_as_built_canceled.sql [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+insert into assets_as_built_investigations
+    (investigation_id       , asset_id)
+values
+    (${investigationSentId6}, ${assetAsBuiltId20});
+
+---
+update assets_as_built
+    set in_investigation = true
+    where id in (${assetAsBuiltId20});
+
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__065_sent_investigation_on_as_built_canceled.sql:41]: Correct, should not be set to true for closed/cancelled alerts and investigations.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__066_sent_investigation_on_as_built_closed.sql [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+-- join investigation to asset
+insert into assets_as_built_investigations
+    (investigation_id       , asset_id)
+values
+    (${investigationSentId7}, ${assetAsBuiltId06});
+
+---
+update assets_as_built
+    set in_investigation = true
+    where id in (${assetAsBuiltId06});
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__066_sent_investigation_on_as_built_closed.sql:40]: Correct, should not be set to true for closed/cancelled alerts and investigations.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__054_received_investigation_on_as_built_closed.sql [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+insert into assets_as_built_investigations
+    (investigation_id           , asset_id)
+values
+    (${investigationReceivedId5}, ${assetAsBuiltId15});
+
+---
+update assets_as_built
+    set in_investigation = true
+    where id in (${assetAsBuiltId15});
+
```
[the remark this answers is not in the document]
Comment (martin-fritz) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__054_received_investigation_on_as_built_closed.sql:41]: Correct, should not be set to true for closed/cancelled alerts and investigations.

**Review by hgo-ds — APPROVED** [review]

File: tx-backend/src/main/resources/db/testdata/afterMigrate__034_received_alert_on_as_built_closed.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (hgo-ds) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__034_received_alert_on_as_built_closed.sql]: We leave it in, setting to `false` to document what the intention is.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__045_sent_alert_on_as_built_canceled.sql (hunk 3 above)
[the remark this answers is not in the document]
Comment (hgo-ds) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__045_sent_alert_on_as_built_canceled.sql]: We leave it in, setting to `false` to document what the intention is.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__054_received_investigation_on_as_built_closed.sql (hunk 7 above)
[the remark this answers is not in the document]
Comment (hgo-ds) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__054_received_investigation_on_as_built_closed.sql]: We leave it in, setting to `false` to document what the intention is.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__066_sent_investigation_on_as_built_closed.sql (hunk 6 above)
[the remark this answers is not in the document]
Comment (hgo-ds) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__066_sent_investigation_on_as_built_closed.sql]: We leave it in, setting to `false` to document what the intention is.

File: tx-backend/src/main/resources/db/testdata/afterMigrate__065_sent_investigation_on_as_built_canceled.sql (hunk 5 above)
[the remark this answers is not in the document]
Comment (hgo-ds) [reply tx-backend/src/main/resources/db/testdata/afterMigrate__065_sent_investigation_on_as_built_canceled.sql]: We leave it in, setting to `false` to document what the intention is.

**Pull request merged**